File: app/src/main/java/com/jakubu9333/bestartists/Artist.java
package com.jakubu9333.bestartists;

import com.jakubu9333.bestartists.database.ArtistEntity;

import java.util.ArrayList;

/**
 * @author Jakub Uhlarik
 */
public class Artist {
    public Artist(String name){
        this.name=name;
    }

    private String name;
    private String uri="xd";
    private ArrayList<Image> images;
    private String url640="xd";
    private String url320;
    private String url160;
    private Boolean chosen = false;

    public String getName() {
        return name;
    }

    public ArtistEntity toEntity(){
        return new ArtistEntity(uri,name,url640);
    }
}

File: app/src/main/java/com/jakubu9333/bestartists/jsonmodels/Artist.kt
package com.jakubu9333.bestartists.jsonmodels

import com.jakubu9333.bestartists.database.ArtistEntity

/**
 *
 * @author Jakub Uhlarik
 */
data class Artist(
    val id:String,
    val name:String,
    val images:List<Image>
){
    fun getDatabaseArtist():ArtistEntity{
        val imgUrl= images[0].url
        return ArtistEntity(id,name,imgUrl)
    }

    override fun toString(): String {
        return "'$name'"
    }


}

